app/src/main/java/org/jellyfin/mobile/ui/screens/connect/ServerSelection.kt

```diff
@@ -333,3 +333,57 @@ private fun ServerDiscoveryItem(
         },
     )
 }
+
+@Preview
+@Composable
+private fun AddressSelectionPreview(
+    @PreviewParameter(AddressSelectionStateProvider::class) state: AddressSelectionPreviewState,
+) {
+    // You can use state.description in the @Preview name if you want dynamic naming,
+    // though Android Studio usually picks up the provider's individual item descriptions.
+    // For explicit naming per preview based on provider values, you'd need multiple @Preview
+    // annotations or a more complex setup if the tooling doesn't pick up names from
+    // a field in the provided state object directly for the preview group name.
+    // However, the individual previews generated by the provider will often show the
+    // `toString()` of the provided parameter, or you can see them in the Preview panel.
+
+    AppTheme {
+        AddressSelection(
+            url = state.url,
+            errorText = state.errorText,
+            loading = state.loading,
+            onTextChange = {},
+            onDiscoveryClick = {},
+            onSubmit = {},
+        )
+    }
+}
+
+@Preview
+@Composable
+private fun ServerDiscoveryItemPreview(
+    @PreviewParameter(ServerSuggestionPreviewParameterProvider::class) serverSuggestion: ServerSuggestion,
+) {
+    AppTheme {
+        ServerDiscoveryItem(
+            serverSuggestion = serverSuggestion,
+            onClickServer = {},
+        )
+    }
+}
+
+@Preview
+@Composable
+private fun ServerDiscoveryListPreview(
+    @PreviewParameter(
+        ServerSuggestionListPreviewParameterProvider::class,
+    ) serverSuggestionsList: List<ServerSuggestion>,
```

Comment:
```suggestion
    @PreviewParameter(ServerSuggestionListPreviewParameterProvider::class)
    serverSuggestionsList: List<ServerSuggestion>,
```
This is how I'd generally wrap all parameters like this, with the annotation on a separate line above.